aiohttp_xmlrpc/client.py

```diff
@@ -110,5 +110,11 @@ def method(*args, **kwargs):
 
         return method
 
+    def __aenter__(self):
+        return type(self.client).__aenter__(self.client)
```

Comment:
I feel it's might be shorter and simpler. Like this:
```python
return self.client.__aenter__()
```

Reply:
But that's not how these methods are called by `async with`

Reply:
But my method generates less bytecode and provides similar result.